mixer/cmd/build.go

```diff
@@ -166,31 +169,47 @@ var buildFormatBumpCmd = &cobra.Command{
 	Long:   `Used to create a downstream format bump`,
 	Hidden: true,
 	Run: func(cmd *cobra.Command, args []string) {
+		if buildFlags.newFormat == "" {
+			fail(errors.New("Please supply the next format version with --new-format"))
+		}
 		b, err := builder.NewFromConfig(configFile)
 		if err != nil {
 			fail(err)
 		}
-		cmdToRun := strings.Split("mixer build format-bump new", " ")
-		if err := b.RunCommandInContainer(cmdToRun); err != nil {
-			fail(err)
+		// Silence linter...
+		_ = b
```

Comment:
If you don't use the builder why are you creating one `NewFromConfig`? Just don't create it anymore.

Reply:
Huh you're right.